chain-signatures/node/src/indexer.rs

```diff
@@ -138,7 +157,17 @@ impl Indexer {
 
     /// Check whether the indexer is behind with the latest block height from the chain.
     pub async fn is_behind(&self) -> bool {
-        self.last_updated_timestamp.read().await.elapsed() > self.behind_threshold
+        let network_latest_height = rpc_client::fetch_latest_block_height(&self.rpc_client).await;
+        if let Ok(network_latest_height) = network_latest_height {
+            self.latest_block_height().await
+                < network_latest_height - self.block_height_lag_threshold
+        } else {
+            false
```

Comment:
Should we return `true` if we fail to get the latest block height?

Reply:
yeah should be `true` if it's behind